subroom.lua

```diff
@@ -150,7 +150,7 @@ function ToArrival( tUser, sMessage )
 			return true
 		end
 	elseif sCmd == "mod" or sCmd == "police" and ('01'):find( tostring(tUser.iProfile) ) then
```

Comment:
Instead of checking whether user profile is `0` or `1`; put a check whether `tUser.bOperator` is true or not.